collections/_tutorials/2023-05-09-tutorial-score-based-models.md

```diff
@@ -0,0 +1,421 @@
+
+The new training objective is a weighted sum of Fisher divergences for all noise scales :
+
+$$ \sum_{i=1}^{L} \lambda (i) \mathbb{E}_{x \sim p(x)} \mathbb{E}_{\tilde{x}\sim p_\sigma(\tilde{x} \vert x)} \left[ \left\Vert \sigma_i s_{\theta}(x,\sigma_i) + \frac{\tilde{x} - x}{\sigma_i} \right\Vert _2^2 \right] $$
+
+With this new way of doing, a T-step Langevin dynamics sampling is decomposed in L consecutive substeps of $$T/L$$ iterations performed with decreasing noise scaling (*i.e.* from $$i=L$$ to $$i=1$$). This is called *annealed Langevin dynamics* (Fig. 5, sampling step are illustrated from right to left).
+
+&nbsp;
+
+<div style="text-align:center">
+<img src="/collections/images/score_based/annealed_langevin_dynamic.gif" width=750></div>
+<p style="text-align: center;font-style:italic">Figure 5. Annealed Langevin Dynamics.</p>
+
+&nbsp;
+
+#### Examples of results
+
+In 2021, a score-based model established new state-of-the-art performances for image generation on CIFAR-10 with an Inception score of 9.89 and FID of 2.20, beating previous best model StyleGAN2-ADA.
+
+
+<div style="text-align:center">
+<img src="/collections/images/score_based/example_cifar10.jpg" width=700></div>
+<p style="text-align: center;font-style:italic">Figure 6. Example of image generated with a score-based model trained on CIFAR-10.</p>
+
+&nbsp;
+
+<div style="text-align:center">
+<img src="/collections/images/score_based/example_faces.jpg" width=700></div>
+<p style="text-align: center;font-style:italic">Figure 7. Examples of samples generated from a score-based model trained on CelebA-HQ (1024x1024).</p>
+
+&nbsp;
+
+### **Quick reminder on diffusion models**
+
+A diffusion model [^6] [^7] [^8] consists in converting form a known, simple and tractable distribution $$ \pi(x)$$ (typically a standard Gaussian) to a target distribution $$p_{data}(x)$$ using a Markov chain.
+
+The forward process consists in corrupting an input image $$x_0$$ via a T-step Markov chain where at each step, a small amount of noise is added to the image. Given an image $$x_t$$ at time step $$t$$, we produce the next image with a Gaussiance noise with variance $$\beta_t$$ :
+
+$$ q(x_{t+1}|x_t)=\mathcal{N}(x_{t+1}, \mu_t = \sqrt{1-\beta_t}x_t, \Sigma_t = \beta_t \textbf{I})$$
+
+More practically, a 'slightly more noisy image' is generated (at a pixel level) via :
+
+$$ x_{t+1} = \sqrt{1-\beta_t}x_t + \sqrt{\beta_t}\epsilon$$
+
+with $$\epsilon \sim \mathcal{N}(0,1)$$
+
+> The total number a diffusion steps T, as well as the noising schedule $$\beta_t$$ must be specified. T is typically of several thousands and $$\beta_t$$ variates linearly from $$\beta_1 = 10^{-4}$$ to $$\beta_T = 0.02$$ in CITER DDPM, even though other schedule have proven to be more efficient since then.
```

Comment:
Typo:
```suggestion
> The total number of diffusion steps T, as well as the noising schedule $$\beta_t$$ must be specified. T is typically of several thousands and $$\beta_t$$ variates linearly from $$\beta_1 = 10^{-4}$$ to $$\beta_T = 0.02$$ in CITER DDPM, even though other schedule have proven to be more efficient since then.
```